src/Runner.jl

```diff
@@ -1284,10 +1286,12 @@ function platform_envs(platform::AbstractPlatform, src_name::AbstractString;
     if libc(platform) === "glibc"
         # This should be kept in sync with the version of glibc used in
         # https://github.com/JuliaPackaging/Yggdrasil/blob/master/0_RootFS/gcc_sources.jl
-        if arch(platform) in ("armv7l", "aarch64")
+        if arch(platform) in ["armv7l", "aarch64"]
```

Comment:
Why this change? Tuples are slightly more efficient as they don't allocate memory at all.

Reply:
Arrays are used in this file in so many places... Really, this kind of efficiency isn't needed here.

Changed to tuples.